cli.js

```diff
@@ -47,7 +56,7 @@ process.stdin.on('keypress', (ch, key) => {
 
 	if (key.name === 'escape' || (key.ctrl && key.name === 'c')) {
 		if (query.length <= 1) {
-			logUpdate();
+			logUpdate(`${pre}${chalk.dim('Hey! You were supposed to use me')}\n\n`);
```

Comment:
But it only cleans it up if user hasn't typed anything. So, why not to display something instead of cleaning?  What you think? :smiley: